src/platforms/javascript/common/session-replay/privacy.mdx

```diff
@@ -51,6 +51,26 @@ Additionally, Sentry attempts to [scrub](/product/data-management-settings/scrub
 
 More details about this feature can be found in the [configuration page](/platforms/javascript/session-replay/configuration/#network-details).
 
+## Custom scrubbing
+
+As of SDK version 7.53.0, `beforeAddRecordingEvent` was added that allows you to scrub and/or filter out recording events before they leave the browser. These events include console logs and network request and response data.
+
+```javascript
+new Sentry.Replay({
+  beforeAddRecordingEvent: (event) => {
+    // Filter out specific events
+    if (event.data.tag === "foo") {
+      return null;
+    }
+
+    // Remember to return an event if you want to keep it!
+    return event;
+  },
+});
+```
+
+Note that we have server-side scrubbing for this data that will certain patterns such as American social security numbers, credit cards, and private keys.
```

Comment:
```suggestion
Note that we have [server-side scrubbing](/product/data-management-settings/scrubbing/server-side-scrubbing/) for this data that will certain patterns such as American social security numbers, credit cards, and private keys.
```

Reply:
Building on the previous suggestion by adding "PII" in there:
```suggestion
Note that we have [server-side PII scrubbing](/product/data-management-settings/scrubbing/server-side-scrubbing/) for this data that will certain patterns such as American social security numbers, credit cards, and private keys.
```